website/templates/emails/pending_embargo_admin.html.mako

```diff
@@ -6,22 +6,44 @@
     Hello ${user.fullname},<br>
     <br>
     % if is_initiator:
-    You initiated an embargoed registration of ${project_name}. The proposed registration can be viewed here: ${registration_link}.<br>
+        % if is_moderated:
```

Comment:
This would have been much easier to read and maintain if you'd done a top-level `if` on `is_moderated`, so that the structure of the template matched the structure of the document that described it.  Having several serial `is_moderated` if statements makes it confusing to read.

Reply:
I combine the is_moderated conditionals bu always including the embargo date like we mentioned the earlier disccussion.